frontend/app/component/form/data_entry/candidates_votes/CandidatesVotesForm.test.tsx

```diff
@@ -260,10 +264,6 @@ describe("Test CandidatesVotesForm", () => {
       const submitButton = screen.getByRole("button", { name: "Volgende" });
       await user.click(submitButton);
 
-      const expectedValidFields = [candidateVotes0, candidateVotes1, total] as HTMLElement[];
-      expectFieldsToBeValidAndToNotHaveAccessibleErrorMessage(expectedValidFields);
-      expectFieldsToNotHaveIcon(expectedValidFields);
-
```

Comment:
Did you remove the fields checks here on purpose?

Reply:
I did, because I don't think we should check this in a test about the API request being correct.